includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -257,10 +257,14 @@ public function video_override( $html, $attr ) {
 		}
 		$src             = $attr['src'];
 		$youtube_pattern = '#^https?://(?:www\.)?(?:youtube\.com/watch|youtu\.be/)#';
-		if ( 1 === preg_match( $youtube_pattern, $src ) ) {
-			return $this->shortcode( [ $src ] );
```

Comment:
I moved the small amount of logic from `shortcode()` that this used into this function, so `shortcode()` can be `@deprecated`.

Reply:
Can't we just remove it altogether? If methods aren't being referenced anymore, then we should try to remove them altogether.